src/libraries/ReceiverWeights.sol

```diff
@@ -79,65 +74,36 @@ library ReceiverWeightsImpl {
         returns (
             address receiver,
             address receiverHint,
-            uint32 weightReceiver,
-            uint32 weightProxy
+            uint32 weight
         )
     {
         receiver = (prevReceiver == ADDR_ROOT) ? self.data[ADDR_ROOT].next : prevReceiverHint;
         while (receiver != ADDR_ROOT) {
-            weightReceiver = self.data[receiver].weightReceiver;
-            weightProxy = self.data[receiver].weightProxy;
+            weight = self.data[receiver].weight;
             receiverHint = self.data[receiver].next;
-            if (weightReceiver != 0 || weightProxy != 0) break;
+            if (weight != 0) break;
             receiver = receiverHint;
         }
     }
 
-    /// @notice Checks if the list is fully zeroed and takes no storage space.
-    /// It means that either it was never used or that
-    /// it's been pruned after removal of all the elements.
-    /// @return True if the list is zeroed
-    function isZeroed(ReceiverWeights storage self) internal view returns (bool) {
-        return self.data[ADDR_ROOT].next == ADDR_ROOT;
-    }
-
     /// @notice Set weight for a specific receiver
     /// @param receiver The receiver to set weight
     /// @param weight The weight to set
     /// @return previousWeight The previously set weight, may be zero
-    function setReceiverWeight(
+    function setWeight(
```

Comment:
```suggestion
    function _setWeight(
```

Maybe lets change it for consistency in the library as well.

Reply:
I'm not sure about that, `ReceiverWeights` is a library and all its functions are `internal`. Should every function in every library be prefixed with an underscore?